Update Custom HTTP Routes Documentation (#1136)
slackapi/bolt-js: docs/_advanced/custom_routes.md

---

<div class="section-content">

> As of `v3.7.0`, custom HTTP routes can be easily added by passing in a collection of routes as `customRoutes` when initializing `App`. 

Reviewer: ```suggestion
As of `v3.7.0`, custom HTTP routes can be easily added by passing in an array of routes as `customRoutes` when initializing `App`. 
```

Reviewer: Nit: We could be more specific here since keyed collections like Maps are not accepted.

Each `CustomRoute` object must contain three properties: `path`, `method`, and `handler`. `method`, which corresponds to the HTTP verb, can be either a string or an array of strings.
</div>

```javascript
const { App } = require('@slack/bolt');

> // Create the Bolt App, using the receiver

Reviewer: ```suggestion
// Initialize Bolt app, using the default HTTPReceiver
```

Author: Oops, this probably shouldn't even be there. This was carry over from the existing example.
